redbot/core/cog_manager.py

```diff
@@ -322,11 +323,15 @@ class CogManagerUI:
         """
         Lists current cog paths in order of priority.
         """
-        install_path = await ctx.bot.cog_mgr.install_path()
-        cog_paths = await ctx.bot.cog_mgr.paths()
-        cog_paths = [p for p in cog_paths if p != install_path]
-
-        msg = _("Install Path: {}\n\n").format(install_path)
+        cog_mgr = ctx.bot.cog_mgr
+        install_path = await cog_mgr.install_path()
+        core_path = cog_mgr.CORE_PATH
+        cog_paths = await cog_mgr.paths()
+        cog_paths = [p for p in cog_paths if p not in (install_path, core_path)]
+
+        msg = _("Install Path: {install_path}\nCore Path: {core_path}\n\n").format(
+            install_path=install_path, core_path=core_path
+        )
```

Comment:
I've made the core path appear separately to the rest of the paths, to show it's not set in Config.